app/__init__.py

```diff
@@ -0,0 +1,71 @@
+import os
+from flask import Flask, jsonify, g
+from flask.ext.sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+
+def create_app(config_name):
```

Comment:
Default `config_name` so it'll work even when instantiating the app through a different method. For instance using uwsgi instead of run.py.